kyuubi-main/src/test/scala/org/apache/kyuubi/operation/SharedKyuubiOperationContext.scala

```diff
@@ -17,13 +17,13 @@
 
 package org.apache.kyuubi.operation
 
-import org.apache.kyuubi.Utils
+import org.apache.kyuubi.{KyuubiFunSuite, Utils}
 import org.apache.kyuubi.config.KyuubiConf
-import org.apache.kyuubi.ha.HighAvailabilityConf._
+import org.apache.kyuubi.ha.HighAvailabilityConf.{HA_ZK_ACL_ENABLED, HA_ZK_QUORUM}
 import org.apache.kyuubi.ha.server.EmbeddedZkServer
 import org.apache.kyuubi.server.KyuubiServer
 
-abstract class KyuubiOperationSuite extends JDBCTests {
+trait SharedKyuubiOperationContext extends KyuubiFunSuite {
```

Comment:
For sense of symmetry, `WithKyuubiServer` as we have a `WithSparkSQLEngine` 😺

Reply:
👌